scripts/run_linter.sh

```diff
@@ -68,13 +68,13 @@ if [ ! -d "$TOOLS_DIR/pycodestyle-2.3.1" ]; then
 fi
 
 if [ "$TRAVIS" == 'true' ]; then
-  pycodestyle -v --exclude=./node_modules,.git,./.vscode,./.circleci || exit 1
-  pydocstyle -v || exit 1
+  pydocstyle --match-dir="^(?!oppia_tools|\.vscode).*" -v || exit 1
+  pycodestyle -v --exclude=./node_modules,.git,./.vscode,./.circleci,./oppia_tools,./third_party || exit 1
   pylint_runner -v || exit 1
 else
   # These commands might not work cross-platform.
-  $PYTHON_CMD $TOOLS_DIR/pydocstyle-2.1.1/src/pydocstyle/__main__.py -v || exit 1
-  $PYTHON_CMD $TOOLS_DIR/pycodestyle-2.3.1/pycodestyle.py -v --exclude=./node_modules,.git,./.vscode,./.circleci || exit 1
+  $PYTHON_CMD $TOOLS_DIR/pydocstyle-2.1.1/src/pydocstyle/__main__.py --match-dir="^(?!oppia_tools|\.vscode).*" -v || exit 1
```

Comment:
Same thing. Actually you could reduce the branching here perhaps? I.e. store the command in a variable that's set in the if/else branches, and then, after the branching, run the command with the relevant args.

Reply:
Done! PTAL again (for file `run_linter.sh`)